core/api.go

```diff
@@ -88,8 +98,9 @@ func (a *api) Start() {
 		threads.GET("/:id", a.getThreads)
 		threads.DELETE("/:id", a.rmThreads)
 		threads.POST("/:id/files", a.addThreadFiles)
+		threads.GET("/:id/updates", a.streamThreads)
```

Comment:
does this seem like a reasonable endpoint?

Reply:
lgtm